CL/cl_ext_intel.h

```diff
@@ -628,6 +629,24 @@ clEnqueueMemcpyINTEL_fn)(
             const cl_event* event_wait_list,
             cl_event* event);
 
+extern CL_API_ENTRY cl_mem CL_API_CALL
+clCreateBufferWithPropertiesINTEL(
```

Comment:
Since this extension adds new APIs (and not just new enums, though see #86), it should add an extension `#define`:

```c
#define cl_intel_create_buffer_with_properties 1
```